src/styles/header.scss

```diff
@@ -0,0 +1,58 @@
+@import 'variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  .logo {
+    display: flex;
+  }
+
+  .nav__list {
+    list-style-type: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+
+    .nav__item {
+      margin-right: 20px;
+      position: relative;
+
+      &:last-child {
+        margin-right: 0;
+      }
+
+      .nav__link {
+        display: flex;
+        align-items: center;
+        height: 60px;
+        text-decoration: none;
+        color: $main-color;
+        font-size: 12px;
+        font-weight: 500;
+        text-transform: uppercase;
```

Comment:
Font size should be set in `px`, `em`, or `rem` rather than `pt` to ensure better scaling across devices. Also, check if `12px` is a multiple of 4, and if not, adjust it to follow the code style guide.